Simplified/AxisService/AxisService.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  AxisService.swift
+//  Simplified
+//
+//  Created by Raman Singh on 2021-04-01.
+//  Copyright © 2021 NYPL Labs. All rights reserved.
+//
+
+import Foundation
+
+#if FEATURE_DRM_CONNECTOR && OPENEBOOKS
```

Comment:
the `FEATURE_DRM_CONNECTOR` macro is used for Adobe, so it would be better to use a different name.